### DISCUSSION TITLE
stat-computing-r-python/part-advanced-topics/06-interactive-graphics

### DISCUSSION THREAD

**Discussion by giscus** [srvanderplas/stat-computing-r-python#22] [General]
# stat-computing-r-python/part-advanced-topics/06-interactive-graphics

https://srvanderplas.github.io/stat-computing-r-python/part-advanced-topics/06-interactive-graphics.html

**Comment by Sgraham12** [comment]
The written tutorial link is broken, I think this works https://shiny.posit.co/r/getstarted/shiny-basics/lesson1/index.html

**Reply by srvanderplas** [reply]
Fixed, thanks! Just have to get my computer to build the thing locally and I'll push.
